### PR TITLE
Added Form Validation, CreateCustomerSchema, Customer Search, Input, Select and Label Components

### PR DESCRIPTION
**Pull request by harmlessprince**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,13 +16,15 @@
     "core-js": "^3.8.3",
     "pinia": "^2.0.28",
     "register-service-worker": "^1.7.1",
+    "vee-validate": "^4.8.3",
```
Comment (teebarg) [inline package.json:19]: @harmlessprince @cheedinmar do we need this additional library?

File: package.json [hunk 2]
```diff
@@ -16,13 +16,15 @@
...
     "vue": "^3.2.45",
     "vue-currency-input": "^3.0.3",
     "vue-paystack": "^2.0.4",
     "vue-router": "^4.1.6",
     "vue3-otp-input": "^0.3.6",
     "vue3-paystack": "^0.0.5",
-    "vuex": "^4.0.2"
+    "vuex": "^4.0.2",
+    "yup": "^1.0.2"
```
Comment (teebarg) [inline package.json:27]: Why do we need `yup`

File: src/views/Home.vue [hunk 3]
```diff
@@ -162,58 +182,28 @@ import DashboardHeader from "@/components/DashboardHeader.vue";
...
+async function FetchDashboard() {
+    await Apis.dashboarddata().then((res) => {
+        const summary = [
+            { name: "No of Sales", href: "#", icon: ScaleIcon, amount: res?.data?.result?.total_number_of_sales },
+            { name: "Total Revenue", href: "#", icon: ScaleIcon, amount: formatCurrency(res?.data?.result?.total_revenue) },
+            { name: "Commission", href: "#", icon: ScaleIcon, amount: "₦25,659.45" },
+        ];
+        cards.value.push(...summary);
+        transactions.value.push(...res.data.result.recent_activities);
+        console.log()
```
Comment (teebarg) [inline src/views/Home.vue:202]: remove log

File: src/views/CreateCustomer.vue [hunk 4]
```diff
@@ -1,202 +1,241 @@
...
+import { ref, watch } from "vue";
+import { Form } from "vee-validate";
+import AppInput from "../components/AppInput.vue";
+import AppLabel from "../components/AppLabel.vue";
+import AppSelectInput from "../components/AppSelectInput.vue";
+import plus from "@/assets/svgs/plus.vue";
+import defaultButton from "@/components/button.vue";
 import App from "@/layouts/App.vue";
-import { handleError } from "../utilities/GlobalFunctions"
+// import { handleError } from "../utilities/GlobalFunctions";
```
Comment (cheedinmar) [inline src/views/CreateCustomer.vue:128]: remove comment

**Review by teebarg — APPROVED** [review]

**Pull request merged**